packages/api/src/index.ts

```diff
@@ -1,5 +1,54 @@
+import 'cross-fetch/polyfill';
+
+import { API_ENDPOINT } from './constants';
+import { HTTPError } from './httpError';
+import { IApiOptions, IFetchOptions, IGraphQLRequest, IGraphQLResponse, IGraphQLVariables } from './types';
+
 export class Api {
-  constructor() {
-    console.log('api instance has created');
+  private readonly url: string;
+  private readonly headers: IApiOptions['headers'];
+
+  constructor(options: IApiOptions) {
+    const { workspaceId, headers } = options;
+
+    this.url = `${API_ENDPOINT}/${workspaceId}`;
+    this.headers = headers;
+  }
+
+  public async request<T>(
+    query: string,
+    variables?: IGraphQLVariables,
+    options: IFetchOptions = { headers: {} },
+  ): Promise<IGraphQLResponse<T>> {
```

Comment:
typo with double ">>"?

Reply:
No, this isn't a typo. There is double generic usage.